### PR TITLE
fix(arpa_reporting): add main and h1 elements - axe-dev finding

### PR DESCRIPTION
**Pull request by lsr-explore**
## Description
Address color contrast and structure issues found by axe devTools scan

## Before 
image

## After - no issues flagged
image

### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [ ] Provided ticket and description
- [ ] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: packages/client/src/arpa_reporter/components/PageNavigation.vue [hunk 1]
```diff
@@ -70,98 +70,99 @@
...
-          to="/uploads"
-        >
-          Uploads
-        </router-link>
-      </li>
-
-      <li
-        v-if="role === 'admin'"
-        class="nav-item"
+    <main>
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/components/PageNavigation.vue:73]: Note to reviewer:  Check hide whitespace to review - only change was to add <main></main> tag.

File: packages/client/src/arpa_reporter/scss/index.scss [hunk 2]
```diff
@@ -1,22 +1,123 @@
...
+  color: $negative-default;
+}
+
+.color-green {
+  color: $positive-default;
+}
+
+.usdr-tab-nav-link {
+  color: #305CE0;
+}
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/scss/index.scss:91]: Note to reviewer:

As we don't plan to continue developing the tool, I created some classes to override the bootstrap colors as a quick fix.

Note:  [Bootstrap page on accessibility](https://getbootstrap.com/docs/4.0/getting-started/accessibility/)

> Color contrast
Most colors that currently make up Bootstrap’s default palette—used throughout the framework for things such as button variations, alert variations, form validation indicators—lead to insufficient color contrast (below the recommended [WCAG 2.0 color contrast ratio of 4.5:1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)) when used against a light background. Authors will need to manually modify/extend these default colors to ensure adequate color contrast ratios.

File: packages/client/src/arpa_reporter/scss/colors-base-tokens.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/**
+ * colors-base-tokens.scss
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/scss/colors-base-tokens.scss:2]: Note to reviewer:  Copied these from the grants scss for parity

File: packages/client/src/arpa_reporter/scss/colors-semantic-tokens.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
+/**
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/scss/colors-semantic-tokens.scss:1]: Note to reviewer:  Copied these files from the grants scss for parity

File: packages/client/src/arpa_reporter/scss/fonts.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
+/**
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/scss/fonts.scss:1]: Note to reviewer:  Copied these files from the grants scss for parity

File: packages/client/src/arpa_reporter/scss/index.scss [hunk 6]
```diff
@@ -1,22 +1,123 @@
-// override theme variables here
+/**
```
Comment (lsr-explore) [inline packages/client/src/arpa_reporter/scss/index.scss:1]: Note to reviewer:  Copied these files from the grants scss for parity

**Review by TylerHendrickson — APPROVED** [review]
@lsr-explore Changes make sense to me & look good locally – approved! 💯

**Review by TylerHendrickson — APPROVED** [review]

**Pull request merged**